compute/admin_guide/configure/configure-agentless-scanning.adoc

```diff
@@ -12,7 +12,31 @@ ifdef::compute_edition[]
 
 * Ensure you have permissions to create service keys and security groups in your cloud account.
 * Ensure you have permissions to apply agentless permission templates to your cloud account.
-* Ensure you can connect to the Prisma Cloud Console over HTTPS from your cloud account. If default security group is not available, create custom security group with custom VPC that allows connection for scanners from the account to Prisma Cloud Console.  
+* Ensure you can connect to the Prisma Cloud Console over HTTPS from your cloud account. If default security group is not available, create custom security group with custom VPC that allows connection for scanners from the account to Prisma Cloud Console. A recommended security group should deny all incoming traffic and allow one egress port only which is tcp/443 for the SaaS and tcp/8083 for the self-hosted console.
```

Comment:
I think it's good to mention the key characteristics that the security group should have, please accept my suggestion to apply the needed changes.
```suggestion
* Ensure you can connect to the Prisma Cloud Console over HTTPS from your cloud account. If default security group is not available, create custom security group with custom VPC that allows connection for scanners from the account to Prisma Cloud Console. 
Ideally, your security group denies all incoming traffic and allows one egress port. 
The egress port is either `tcp/443` for the Prisma Cloud Enterprise Editon or `tcp/8083` for the Prisma Cloud Compute Edition.
```